common/lib/xmodule/xmodule/seq_module.py

```diff
@@ -654,7 +641,8 @@ def _render_student_view_for_items(self, context, display_items, fragment, view=
             bookmarks_service = self.runtime.service(self, 'bookmarks')
         except NoSuchServiceError:
             bookmarks_service = None
-        context['username'] = self.runtime.service(self, 'user').get_current_user().opt_attrs.get(
+        user = self.runtime.service(self, 'user').get_current_user()
+        context['username'] = user.opt_attrs.get(
```

Comment:
Any reason for this change? It doesn't look like the `user` variable ends up being used anywhere else.

Reply:
had initially used it somewhere else but i thought i'd leave it since this might be a bit more readable